opencensus/trace/internal/local_span_store_test.cc

```diff
@@ -25,10 +25,10 @@ namespace trace {
 class SpanTestPeer {
  public:
   static void End(absl::Duration latency, Span* span) {
+    // Call Span::End to get into stores.
+    span->End();
```

Comment:
Won't this potentially store the span into the wrong latency bucket since latency hasn't been set yet?

Reply:
Fixed now.